cursor/cursor.go

```diff
@@ -138,6 +138,14 @@ func (m Model) Mode() Mode {
 //
 // For available cursor modes, see type CursorMode.
 func (m *Model) SetMode(mode Mode) tea.Cmd {
+	// Adjust the mode value if it's less than 0.
```

Comment:
I think if the mode is invalid we should simply `break` instead of changing the mode.

Reply:
Oh yes,  so it will keep the state variable of the model as it is.

Reply:
Yep, exactly!

Reply:
Done